src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -42,7 +42,11 @@
         'list.example': 'Example'|trans({}, 'Admin.Global'),
         'list.edit': 'Edit symbol / format'|trans({}, 'Admin.International.Feature'),
         'list.reset': 'Reset settings'|trans({}, 'Admin.Actions'),
-        'list.example.format': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature')
+        'list.example.format': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature'),
+        'confirmTitle': 'Restore default settings'|trans({}, 'Admin.International.Feature'),
```

Comment:
Please use the same format for translation keys, something like `modal.confirm.title`, `modal.confirm.restore`, `modal.confirm.message`, ...
And `closeButtonLabel` is redundant with the already present `modal.cancel`